specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/examples/createOrUpdateGuestDiagnosticSettings.json

```diff
@@ -0,0 +1,241 @@
+{
+    "parameters": {
+        "subscriptionId": "187f412d-1758-44d9-b052-169e2564721d",
+        "resourceGroupName": "Default-ResourceGroup",
+        "diagnosticSettingsName": "SampleDiagSetting",
+        "api-version": "2018-06-01-preview",
+        "diagnosticSettings": {
+            "location": "Global",
+            "tags": {},
+            "properties": {
+                "osType":  "windows",
+                "dataSources": [
+                    {
+                        "kind": "PerformanceCounter",
+                        "configuration": {
+                            "perfCounters": [
+                                {
+                                    "name": "\\Process(_Total)\\%Processor Time",
+                                    "samplingPeriod": "PT1M"
+                                },
+                                {
+                                    "name": "\\Process(_Total)\\Working Set",
+                                    "samplingPeriod": "PT1M"                                
+                                }
+                            ]
+                        },
+                        "sinks": [
+                            {
+                                "kind": "LogAnalytics"
```

Comment:
Don't you need the details for the kind? The workspace resource ID at least?

Reply:
Not for this private preview. Remember - multi-homing is not supported. Workspace resource id would be passed by extension